### ISSUE TITLE
Subscribe Button

### ISSUE THREAD

**Issue by coke11** [coke11/yoshi123#7]
We need to get the Subscribe button to line up with the intro video. Here is a picture of what Yoshi 123/ Caleb/ Yoshi wants.

![img_2375 2](https://cloud.githubusercontent.com/assets/18141280/16904477/e76bc5b0-4c5c-11e6-950a-c137fdd49659.jpeg)

Where it says "Here" he would like the button. He says the button could also be centered below, or beside the video.

This is mostly just a note for myself, but if someone would like to help me or submit a pull request on this, that would be great. 

Thanks
coke

**Comment by joker314** [comment]
To centre it below the video you could move it after a `<br/>` but before the `</center>` tag. To put it beside might take a bit f research but it can also be done

**Comment by coke11** [comment]
Yes. I knew the centering part. But he wants it beside. Maybe a table?

**Comment by joker314** [comment]
You are a genius! But the centring of the video wouldn't work aymore, hmm..

**Comment by joker314** [comment]
I think the table is the best way to go, if you can get it to work,,

**Comment by coke11** [comment]
Ok :). If I can find some time I will try to work on that...

**Comment by joker314** [comment]
Ok :+1:

**Comment by csf30816** [comment]
Vertical centering maybe? There are different ways of doing that, I would have to know more about both elements. But on phone now.
